source/code/plugins/out_oms.rb

```diff
@@ -91,8 +91,9 @@ def write(chunk)
       datatypes = {}
       unmergable_records = []
       chunk.msgpack_each {|(tag, record)|
-        if record.has_key?('DataType') and record.has_key?('IPName')
-          key = "#{record['DataType']}.#{record['IPName']}".upcase
+        if record.has_key?('DataType')
```

Comment:
if this change is for the ADM issue .. shouldn't we instead fix the case-sensitivity issue? i.e. accept both IPName and IPNAME .. but I still think we should have exactly one of those 2